graf2d/gpadv7/v7/inc/ROOT/RAttrLine.hxx

```diff
@@ -42,6 +42,48 @@ public:
    int GetStyle() const { return Get<int>("style"); }
 };
 
+
+class RAttrLineNew : public RAttributesVisitor {
+
+   auto &getDefaults()
+   {
+      static RDrawableAttributes::Map_t dflts;
+      if (dflts.empty()) {
+         dflts["width"] = std::make_unique<RDrawableAttributes::DoubleValue_t>(1);
```

Comment:
The defaults must not be provided in code but in the style file, else users will be confused about "what's the default", and / or we'd have to keep things in sync (and fail).

Reply:
Idea here - if none of normal attributes sources (attributes container or RStyle) does not provide any value, one has something. For instance, if we want to force default values for current object.